sound/soc/codecs/sdw-null.c

```diff
@@ -0,0 +1,304 @@
+// SPDX-License-Identifier: GPL-2.0-only
+//
+// sdw-null.c -- a dummy SoundWire codec for data transfer tests
+//
+// Copyright(c) 2021 Intel Corporation
+//
+//
```

Comment:
I think only the "SPDX-License" line has to be a `//`-comment, the rest should be a normal kernel multi-line `/* */` comment

Reply:
there isn't really any consistency that I can understand

````
sound/soc/codecs$ grep '// Copyright' *.c | wc -l
90
sound/soc/codecs$ grep '* Copyright' *.c | wc -l
260
sound/soc/codecs$ grep '* Copyright' *.h | wc -l
180
sound/soc/codecs$ grep '// Copyright' *.h | wc -l
5
````

Reply:
answer from Mark Brown: "The whole block in C++ style please."